code/modules/power/cell.dm

```diff
@@ -107,8 +107,8 @@
 	message_admins("LOG: Rigged power cell explosion, last touched by [fingerprintslast]")
 
 	explosion(T, devastation_range, heavy_impact_range, light_impact_range, flash_range)
-
-	spawn(1)
+	if(!gcDestroyed)
```

Comment:
I'm not sure this check needs to be here.
If we get rid of it, charge is set to 0, and the `if (charge==0)` check will cause it to return instantly anyway, so it won't qdel itself again